ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -45,13 +45,15 @@ import scala.concurrent.Future
 import scala.util.Try
 import scala.util.control.NonFatal
 
-private class PostgresLedgerDao(
+private class JdbcLedgerDao(
     dbDispatcher: DbDispatcher,
     contractSerializer: ContractSerializer,
     transactionSerializer: TransactionSerializer,
     valueSerializer: ValueSerializer,
-    keyHasher: KeyHasher)
+    keyHasher: KeyHasher,
+    dbType: JdbcLedgerDao.DbType)
```

Comment:
I'm a bit doubtful about this approach. I would rather use different implementations of some holder of queries for specific implementations rather than manually dispatching with cases.

Reply:
Conversation continued below